.husky/commit-msg

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-npx --no-install commitlint --edit
+npx --no -- commitlint --edit ${1}
```

Comment:
According to the docs: https://commitlint.js.org/#/guides-local-setup?id=add-hook